public/content/lessons/2017/derivative-formulas-geometrically/index.mdx

```diff
@@ -1,10 +1,381 @@
+
+<Figure
+  image="figures/8.11-9.35-base-cases.svg"
+/>
+
+However, from a geometric perspective, we have hit a roadblock. How do we visualize four dimensions? One path forward is to continue with algebra, using our geometric intuition to inform the base cases. For example, we can expand higher degree monomial expressions, where the input $x$ have been nudged by a small amount $dx$. This gives us the following series of expressions.
+
+<!-- TODO: possibly expand $f(x) = x^4$ by hand ? -->
+
+| Polynomial | Expansion |
+| --- | --- |
+| $f(x) = x^1$ | $(x + dx)^1 = x + \color{#fc6255}{1} \color{black} dx$ |
+| $f(x) = x^2$ | $(x + dx)^2 = x^2 + \color{#fc6255} 2x \color{black} dx \color{#AAAAAA} + dx^2$ |
+| $f(x) = x^3$ | $(x + dx)^3 = x^3 + \color{#fc6255} 3x^2 \color{black} dx \color{#AAAAAA} + 3xdx^2 + dx^3$ |
+| $f(x) = x^4$ | $(x + dx)^4 = x^4 + \color{#fc6255} 4x^3 \color{black} dx \color{#AAAAAA} + 6x^2dx^2 + 4xdx^3 + dx^4$ |
+| $f(x) = x^5$ | $(x + dx)^5 = x^5 + \color{#fc6255} 5x^4 \color{black} dx \color{#AAAAAA} + 10x^3dx^2 + 10x^2dx^3 + 5xdx^4 + dx^5$ |
+
+
+<!-- | $f(x) = x^6$ | $(x + dx)^6 = x^6 + \color{#fc6255} 6x^5 \color{black} dx \color{#AAAAAA} + 15x^4dx^2 + 20x^3dx^3 + 15x^2dx^4 + 6xdx^5 + dx^6$ | -->
+
+<!-- TODO: link to binomial expansion -->
+
+Focusing on the rate in change introduced by the very small nudge, highlighted in red, and ignoring expressions containing $dx^2$, the pattern that emerges is what's known in the business as the "power rule". Given a monomial function raised to some power, $n$, applying the rule gives us the derivative of the function.
+
+<Figure
+  image="figures/7.32.svg" 
+  caption="Power Rule Definition"
+/>
+
+Even though in practice you will find yourself performing this derivative quickly and symbolically, imagining that exponent hopping down to the front, every now and then it’s nice to step back and remember why this rule works. Not just because it’s pretty, and not just because it helps to remind us that math actually makes sense and isn’t just a pile of formulas to memorize, but because it flexes that very important muscle of thinking about derivatives in terms of tiny nudges.
+
+<Question
+  question="What is the derivative of the function $f(x)= 3x^2$?"
+  choice1="$2x$"
+  choice2="$5x$"
+  choice3="$3x$"
+  choice4="$6x$"
+  answer={4}
+>
+
+This question can be modeled and solved as a question of change in area. For example, nudging the input of the function by $dx$ gives the expression $3(x+d x)^{2}$ which expands to the expression $3(x + dx)( x + dx)$. I've drawn the area below where the $3$ scales one of the sides.
```

Comment:
```suggestion
You certainly don't need to visualize this to answer the question. In fact, in practice, it would be absurd to do so. Nevertheless, continuing with the playful and visual spirit of this lesson, it's fun to think of a few ways to do just that.

This question can be modeled and solved as a question of change in area. For example, nudging the input of the function by $dx$ gives the expression $3(x+d x)^{2}$ which expands to the expression $3(x + dx)( x + dx)$. I've drawn the area below where the $3$ scales one of the sides.
```